coalib/output/ConsoleInteraction.py

```diff
@@ -347,20 +344,18 @@ def print_results_formatted(log_printer,
                                         severity_str=severity_str,
                                         **format_args))
         except KeyError as exception:
-            log_printer.log_exception(
-                'Unable to print the result with the given format string.',
-                exception)
+            logging.exception(
+                'Unable to print the result with the given format string.{}'
+                .format(exception))
```

Comment:
logging.exception implicitly adds the traceback to the log message. See the output of 

```
import logging
try:
    1/0
except Exception as e:
    logging.exception("message")
```